main.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable no-unused-vars */
+const homeLink = document.querySelector('#homeLink');
+const aboutLink = document.querySelector('#aboutLink');
+const speakersCards = document.querySelector('.speakers-elt');
+
+function OpenMenu() {
+  document.querySelector('.menuformobile').style.display = 'flex';
+  document.querySelector('.closeicon').style.display = 'block';
+  document.querySelector('main').style.filter = ' blur(6px)';
+}
+
+function CloseMenu() {
+  document.querySelector('.menuformobile').style.display = 'none';
+  document.querySelector('header').style.filter = ' blur(0)';
+  document.querySelector('main').style.filter = ' blur(0)';
+  document.querySelector('headeline').style.filter = ' blur(0)';
+  document.querySelector('footer').style.filter = ' blur(0)';
+}
+
+aboutLink.addEventListener('click', () => {
+  document.querySelector('.menuformobile').style.display = 'none';
+  document.querySelector('header').style.filter = ' blur(0)';
+  document.querySelector('.headeline').style.filter = ' blur(0)';
+  document.querySelector('main').style.filter = ' blur(0)';
+  document.querySelector('footer').style.filter = ' blur(0)';
+});
+
+homeLink.addEventListener('click', () => {
+  document.querySelector('.menuformobile').style.display = 'none';
+  document.querySelector('header').style.filter = ' blur(0)';
+  document.querySelector('.headeline').style.filter = ' blur(0)';
+  document.querySelector('main').style.filter = ' blur(0)';
+  document.querySelector('footer').style.filter = ' blur(0)';
+});
+
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Please do no disable linter rules. These are here to help us write clean code.